src/Client/Client.fsproj

```diff
@@ -5,10 +5,12 @@
 	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
 		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
 		<WarningsAsErrors />
+		<NoWarn>NU1605</NoWarn>
```

Comment:
NU1605 what is this warning?

Reply:
After latest upgrade, VS2019 started to show "Detected package downgrade" warnings and for us this breaks things because we treat warnings as errors. However, seems like this is no proper solution for that and there are only [workarounds](https://github.com/dotnet/core/issues/907#issuecomment-326092990).

Maybe this will go after we upgrade to .NET Core 3.1.